src/Facebook/FacebookClient.php

```diff
@@ -197,7 +197,7 @@ public function prepareRequestMessage(FacebookRequest $request)
      */
     public function sendRequest(FacebookRequest $request)
     {
-        if (get_class($request) === 'FacebookRequest') {
+        if (get_class($request) === 'Facebook\FacebookRequest') {
```

Comment:
No, of course I don't mind to use instanceof. Why should I? :) 
I just wondering if I should make all changes related to "FacebookBatchRequest" in current pull request.